changelog.txt

```diff
@@ -7,6 +7,7 @@
 * Fix - WooCommerce Payments disappeared from WooCommerce Settings if WooCommerce Subscriptions is activated.
 * Add - REST endpoint to capture payments by order ID
 * Add - Payment request button support for checkout and cart blocks.
+* Fix - Columns are misaligned on Payments->Transactions/Disputes page #1719.
```

Comment:
Discussion to include the issue number in changelog is happening here: https://github.com/Automattic/woocommerce-payments/pull/1741#discussion_r627920439